GitUI/GitUIExtensions.cs

```diff
@@ -112,24 +123,26 @@ void OpenWithDiffTool()
                         : diffOfConflict;
                 }
 
-                if (file.IsSubmodule
-                    && file.GetSubmoduleStatusAsync() is Task<GitSubmoduleStatus> task)
+                var task = file.GetSubmoduleStatusAsync();
+
+                if (file.IsSubmodule && task != null)
```

Comment:
```suggestion
                if (file.IsSubmodule && task is not null)
```

Reply:
I wish there was an analyzer to enforce this.

Reply:
regression...